<launch>
  <node name="serial_node" pkg="rosserial_python" type="serial_node.py" output="screen"/>
  <node name="stepper_test_gui" pkg="scara" type="driver.py" output="screen"/>
</launch>